test/test/smoke/LoadLibrary.java

```diff
@@ -11,8 +11,8 @@
 public class LoadLibrary {
 
     public static void main(String[] args) throws Exception {
-        for (int i = 0; i < 200; i++) {
-            Thread.sleep(10);
+        for (int i = 0; i < 20; i++) {
+            Thread.sleep(100);
```

Comment:
Can you move test changes to a separate PR please?
as they aren't actually related to nightly builds.